plugins/org.eclipse.osee.ats.ide.integration.tests/src/org/eclipse/osee/ats/ide/integration/tests/ats/branch/DemoBranchRegressionTest.java

```diff
@@ -330,6 +333,44 @@ public void testCodeTaskCreationAfterCreateReqArtToDelete() {
       createReqArtToDelExpected.addAll(thirdFourthFifthExpected);
 
       testTasksAgainstExpected(createReqArtToDelExpected);
+
+      testAtsActionsDetailsDerivedRestCall();
+   }
+
+   public void testAtsActionsDetailsDerivedRestCall() {
+      // Test derivedFrom with ATS Id
+      String url = String.format("ats/action/%s/details", codeTeamWf.getAtsId());
+      JsonNode action = testActionRestCall(url, 1);
+      JsonNode derived = action.get("derivedfrom");
+      Assert.assertNotNull(derived);
+      JsonNode derived1 = derived.get(0);
+      Assert.assertEquals(reqTeamWf.getAtsId(), derived1.get("AtsId").asText());
+      Assert.assertEquals(reqTeamWf.getIdString(), derived1.get("id").asText());
+      Assert.assertEquals(reqTeamWf.getName(), derived1.get("name").asText());
+      Assert.assertEquals(reqTeamWf.getArtifactType().toStringWithId(), derived1.get("type").asText());
+      Assert.assertEquals(reqTeamWf.getCurrentStateName(), derived1.get("state").asText());
+
+      // Test derivedFrom with id
+      String url2 = String.format("ats/action/%s/details", codeTeamWf.getIdString());
+      JsonNode action2 = testActionRestCall(url2, 1);
+      JsonNode derived2 = action2.get("derivedfrom");
```

Comment:
Nit: Should this be camel case? "derivedFrom"

Reply:
same